test/e2e/antreaipam_test.go

```diff
@@ -0,0 +1,35 @@
+package e2e
+
+import (
+	"testing"
+)
+
+func TestAntreaIPAM(t *testing.T) {
+	skipIfNotAntreaIPAMTest(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	data.enableAntreaIPAMIfSet(t)
```

Comment:
Here, there's a `skipIfNotAntreaIPAMTest` before `enableAntreaIPAMIfSet` but in other cases, there's only a `enableAntreaIPAMIfSet`. What's the purpose?

Reply:
This will set the ConfigMap, although the detection is not needed because it will be skipped before.